storage/internal/benchmarks/utils.go

```diff
@@ -109,39 +108,23 @@ func generateRandomFile(path string, size int64) (string, error) {
 	return f.Name(), err
 }
 
-// fillDirectoryRandomly fills the directory with up to 1000 different files and
-// subdirectories. Files may be located in any of the subdirectories.
-// The number of bytes in the created files is returned.
-func fillDirectoryRandomly(dirPath string, size int64) (int64, error) {
-	currDirectoryLevel := 0
-	currPath := dirPath
+// fillDirectoryRandomly fills the directory with the number of different files
```

Comment:
Maybe make it explicit that there is only one directory created (no substructure), and change the name to something like `createTempDirectory` (or something else that doesn't include "random".